components/Tabs/src/private/MDCItemBar.h

```diff
@@ -101,6 +101,15 @@
  */
 - (void)itemBar:(nonnull MDCItemBar *)itemBar didSelectItem:(nonnull UITabBarItem *)item;
 
+/**
+ Called before an item is displayed.
+ */
+- (void)itemBar:(nonnull MDCItemBar *)itemBar willDisplayItem:(nonnull UITabBarItem *)item;
```

Comment:
Tip: These shouldn't be necessary - you can just call the tab bar's delegate methods directly.